nfsstat/check.py

```diff
@@ -21,14 +21,14 @@ def __init__(self, name, init_config, agentConfig, instances):
         AgentCheck.__init__(self, name, init_config, agentConfig, instances)
         # if they set the path, use that
         if init_config.get('nfsiostat_path'):
-            self.nfs_cmd = init_config.get('nfsiostat_path')
+            self.nfs_cmd = [init_config.get('nfsiostat_path'), '1']
```

Comment:
Does this change the output at all? Does any of the parsing need to change because of this?

Reply:
Why haven't you added a count? Without it this will run forever without ever reporting metrics.